jina/peapods/pods/k8s.py

```diff
@@ -142,23 +144,31 @@ def start(self) -> 'K8sPod':
 
         :return: self
         """
+        logger = JinaLogger(f'start_{self.name}')
+        logger.info(
+            f'🔑\tCreate Namespace {self.args.k8s_namespace} for {self.name} '
```

Comment:
```suggestion
            f'🏝️\tCreate Namespace {self.args.k8s_namespace} for {self.name} '
```
what about this icon for the namespace? :)

Reply:
SGTM! Cool.